### PR TITLE
Enhance portability by using std::chrono in Timer

### PR DESCRIPTION
**Pull request by wdeconinck**

### CODE REVIEW

File: cmake/tribits.cmake [hunk 1]
```diff
@@ -300,7 +300,9 @@ FUNCTION(TRIBITS_ADD_EXECUTABLE EXE_NAME)
   ENDIF()
 ENDFUNCTION()
 
-ADD_CUSTOM_TARGET(check COMMAND ${CMAKE_CTEST_COMMAND} -VV -C ${CMAKE_CFG_INTDIR})
+IF(NOT TARGET check)
+  ADD_CUSTOM_TARGET(check COMMAND ${CMAKE_CTEST_COMMAND} -VV -C ${CMAKE_CFG_INTDIR})
```
Comment (mhoemmen) [inline cmake/tribits.cmake]: Could you please clarify why you needed to do this?  Thanks!

Comment (wdeconinck) [reply cmake/tribits.cmake]: Certainly. I have a number of CMake projects which I combine into one bundle using "add_subdirectory". It so happens that the target "check" was already defined previous to adding kokkos and in fact with a very similar command. This guard prevents it from being added twice.

Comment (mhoemmen) [reply cmake/tribits.cmake]: @wdeconinck Could you please separate that into a different pull request?  Sorry to be so picky :) .

Comment (wdeconinck) [reply cmake/tribits.cmake]: Okay, no problem. Stay tuned.

**Comment by mhoemmen** [comment]
Thanks @crtrott ! :D

**Pull request merged**